system-tests/services/test-pods.js

```diff
@@ -0,0 +1,256 @@
+require('../_support/utils/ServicesUtil');
+
+describe('Services', function () {
+
+  /**
+   * Test that pods work as expected
+   */
+  describe('Pods', function () {
+
+    beforeEach(function () {
+      cy
+        .visitUrl(`services/overview/%2F${Cypress.env('TEST_UUID')}/create`);
+    });
+
+    it('Create a pod with service address', function () {
+      const serviceName = 'pod-with-service-address';
+      const command = 'python3 -m http.server 8080';
+      const containerImage = 'python:3';
+
+      cy
+        .contains('Multi-container (Pod)')
+        .click();
+
+      cy
+        .root()
```

Comment:
I don't think this `.root` call is needed here or anywhere on this file. It's only needed if you need to reset the context on a chain of commands, but if you're already starting with `cy` it already starts with root.